.github/workflows/test.yaml

```diff
@@ -13,17 +13,11 @@ on:
 jobs:
   build-and-test:
     runs-on: ubuntu-20.04
-    defaults:
-      run:
-        shell: bash -l {0}
-    # This workflow accesses secrets and checks out a PR, so only run if labelled
-    # https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
-    if: contains(github.event.pull_request.labels.*.name, 'preview')
```

Comment:
i think this is left over from snowex hackweek, where we were using NASA URS in notebooks. any time actions that use secrets are triggered from pull requests (which we want to allow for hackweek tutorial developers), it's good to gate the actions with a check from a maintainer, such as adding a label to trigger CI. For now though, we should just run it all the time.

Reply:
I don't like the idea of completely removing this gatekeeping, since it *is* an option we want to keep for tutorial developers. If we don't want to be beholden to it in our template, maybe we should just add an additional if trigger for this organization/repo to always run it?

Reply:
This is also only desirable if we are having jupyterbook *execute* notebooks. Considering it's an advanced configuration I think it's okay to leave this out. We should document it or open an issue though! We still need to use this setup in `netlifypreview.yaml` since credentials are required.

Reply:
I think I figured out my confusion. Anyone submitting a PR would trigger `test.yaml` and any new notebooks would be executed, per `./.github/actions/buildbook` (unless they've all been previously executed and have outputs in the cache). The differences (versus `netlifypreview.yaml`) are this action is just checking to see if it CAN run, but never actually "saves" the run anywhere and we don't use any secrets for the notebook execution part, so it can run publicly/automatically.